templates/bundles/PrestaShopBundle/Admin/macros.html.twig

```diff
@@ -208,7 +208,7 @@
           data-trigger="hover"
           data-container=".{{ popoverContainer }}"
         >
-          <i class="material-icons form-error-icon">error_outline</i> <u>{{ '%count% errors'|transchoice(popoverErrors|length, {}, 'Admin.Global') }}</u>
+          <i class="material-icons form-error-icon">error_outline</i> <u>{{ '%count% errors'|trans({'%count%': popoverErrors|length}, 'Admin.Global') }}</u>
```

Comment:
Does it mean `transchoice` cannot be used anymore?

Reply:
It is deprecated 😄

Reply:
Yeah that's too bad... We just started introducing 2 or 3 plural messages in 178 😅
It's too bad it's a nice feature to have one wording to rule them all, but anyway..

Reply:
@jolelievre @atomiix _The transchoice tag is deprecated since Symfony 4.2 and will be removed in 5.0. Use the ICU MessageFormat with the trans tag instead._

➡️ https://symfony.com/doc/4.4/translation/message_format.html#pluralization